server/services/core_service.go

```diff
@@ -16,17 +17,12 @@ type CoreServiceImpl struct {
 
 // StepGame moves the game state forward using a list of inputs
 func (g *CoreServiceImpl) StepGame(currentState *pogo.GameState, inputs []pogo.GameInputMsg) (*pogo.GameState, error) {
```

Comment:
This bit used to just apply the cards sequentially.  Now it uses the ProcessConditions method to do all of the heavy lifting.